payjoin/tests/integration.rs

```diff
@@ -962,6 +867,18 @@ mod integration {
         payjoin_proposal
     }
 
+    fn extract_pj_tx(
+        sender: &bitcoincore_rpc::Client,
+        psbt: Psbt,
+    ) -> Result<bitcoin::Transaction, Box<dyn std::error::Error>> {
+        let payjoin_psbt = sender.wallet_process_psbt(&psbt.to_string(), None, None, None)?.psbt;
+        let payjoin_psbt = sender.finalize_psbt(&payjoin_psbt, Some(false))?.psbt.unwrap();
+        let payjoin_psbt = Psbt::from_str(&payjoin_psbt)?;
+        tracing::debug!("Sender's Payjoin PSBT: {:#?}", payjoin_psbt);
```

Comment:
we've mixed tracing::debug! and log::debug! in this file which could be unified in a later optimization. No need to stall this huge payoff of tech debt for further optimizations though.